contracts/permit/EIP2612PermitAndDeposit.sol

```diff
@@ -59,7 +59,7 @@ contract EIP2612PermitAndDeposit {
         address _to
     ) internal {
         IERC20(_token).safeTransferFrom(_owner, address(this), _amount);
-        IERC20(_token).safeApprove(_prizePool, _amount);
+        IERC20(_token).safeIncreaseAllowance(_prizePool, _amount);
```

Comment:
Shouldn't we use `safeApprove` for an initial allowance as stated in the library?
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/566a774222707e424896c0c390a84dc3c13bdcb2/contracts/token/ERC20/utils/SafeERC20.sol#L49

Reply:
If you look at the impl it appears to have the same effect